src/main/java/uk/gov/pay/ledger/transaction/state/ExternalTransactionState.java

```diff
@@ -47,13 +55,19 @@ public String getMessage() {
         return message;
     }
 
+    @Schema(example = "true", defaultValue = "null")
+    public Boolean getCanRetry() {
+        return canRetry;
+    }
+
     @Override
     public String toString() {
         return "ExternalTransactionState{" +
                 "value='" + value + '\'' +
                 ", finished=" + finished +
                 ", code='" + code + '\'' +
                 ", message='" + message + '\'' +
+                ", can_retry=" + canRetry +
```

Comment:
This will be null most of the time. Can this be added only when it is non-null?

Reply:
👍